src/conditionalLogicService/evaluateConditionalOptionsPredicate.ts

```diff
@@ -3,23 +3,61 @@ import { FormTypes, ConditionTypes } from '@oneblink/types'
 export default function evaluateConditionalOptionsPredicate({
   predicate,
   predicateElement,
-  predicateValue: value,
+  predicateValue,
 }: {
   predicate: ConditionTypes.ConditionalPredicateOptions
   predicateValue: unknown
   predicateElement: FormTypes.FormElementWithOptions
 }): boolean {
+  /**
+   * In the @oneblink/apps function formService.parseFormElementOptions dynamic
+   * lists are parsed to ensure compliance with One Blink forms. If the dynamic
+   * list options include attributes and the element has attributesMapping
+   * defined then formService.parseFormElementOptions will attempt to match each
+   * dynamic list option attribute value with an option value in the predicate
+   * element's options using the element's attributesMapping. For each attribute
+   * in the element's attrbutesMapping If a predicate option is found for a
+   * particular attribute then the optionIds property of the predicate argument
+   * of this function will be appened with either the matched option's id or
+   * value.
+   *
+   * Eg
+   *
+   * { options: [ { id: 'abc123', label: 'German Shephard',
+   * value:'germanShepherd', attributes: [{ elementId: 'xyz456', optionIds:
+   * ['aaa111']}]}] }
+   *
+   * If a predicate option is not found the optionIds property of the predicate
+   * argument will contain the dynamic list option's attribute value.
+   *
+   * Eg
+   *
+   * { options: [ { id: 'abc123', label: 'German Shephard',
+   * value:'germanShepherd', attributes: [{ elementId: 'xyz456', optionIds:
+   * ['dog']}]}] }
```

Comment:
Can we please wrap these code snippets in ``` so the prettier will format them

Reply:
I tried but it didn't like make a difference. Maybe something up with my prettier config?

Reply:
Probably because it wasn't valid js... I'll update.